doc/howto/network_ovn_setup.md

```diff
@@ -1,33 +1,123 @@
 (network-ovn-setup)=
 # How to set up OVN with LXD
 
+See the following sections for how to set up a basic OVN network, either as a standalone network or to host a small LXD cluster.
+
+## Set up a standalone OVN network
+
+Complete the following steps to create a standalone OVN network that is connected to the parent network (for example, `lxdbr0`) for outbound connectivity.
+
+1. Install the OVN tools on the local server:
+
+        sudo apt install ovn-host ovn-central
+
+1. Configure the OVN integration bridge:
+
+        sudo ovs-vsctl set open_vswitch . \
+          external_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
+          external_ids:ovn-encap-type=geneve \
+          external_ids:ovn-encap-ip=127.0.0.1
+
+1. Create an OVN network:
```

Comment:
Should all of these steps be number one?

Reply:
Yes - Markdown converts them into the correct numbers automatically (even on GitHub ;) ). This is less error-prone than using the real numbers, because you can add or remove steps without changing the numbering.